packages/gatsby-theme-project-portal/gatsby-node.js

```diff
@@ -34,8 +34,54 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   const result = await graphql(`
     query {
       allProject {
+        nodes {
+          title
+          agency
+          topics {
+            title
+          }
+          slug
+          summary
+          statusOfData
+          status
+          startDate
+          requirement
+          question
+          purpose
+          projectTeam {
+            name
+            employer
+          }
+          priorResearch
+          opportunityCloses
+          mainContact {
+            name
+          }
+          fundingInfo
+          expertise
+          faq {
+            text
+            title
+          }
+          deliverable
+          emailContent
+          endDate
+          slug
+        }
+      }
+      allGeneralPage {
         nodes {
           slug
```

Comment:
Looks here like you might have inadvertently removed the slug from the `allCardPage` part of the query.